DSharpPlus/Entities/Channel/Message/DiscordMessage.cs

```diff
@@ -49,7 +49,9 @@ internal DiscordMessage()
             this._stickersLazy = new Lazy<IReadOnlyList<DiscordMessageSticker>>(() => new ReadOnlyCollection<DiscordMessageSticker>(this._stickers));
             this._jumpLink = new Lazy<Uri>(() =>
             {
-                var gid = this.Channel is DiscordDmChannel ? "@me" : this.Channel.GuildId.Value.ToString(CultureInfo.InvariantCulture);
+                var gid = this.Channel != null
```

Comment:
I don't see why channel would ever be null?

Reply:
But even then, I'd prefer you do an if/else or a switch over a nested ternary. They're a pain to read.